fix(bundler/nsis): revert shell context change before deleting appdata, closes #7588 (#7591)
tauri-apps/tauri: .changes/fix-nsis-uninstall-delete-app-data.md

---
'tauri-bundler': 'patch:bug'
---

> On Windows, NSIS uninstall.exe will delete application data in local app data dir of current user if the delete the application data checkbox is checked.

Reviewer: ```suggestion
On Windows, Fix NSIS uninstaller deleting the wrong application data if the delete the application data checkbox is checked.
```
